modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -415,6 +415,31 @@ public String escapeText(String input) {
                         .replace("\"", "\\\""));
     }
 
+    /**
+     * Escape characters while allowing new lines
+     *
+     * @param input String to be escaped
+     * @return escaped string
+     */
+    public String escapeTextWhileAllowingNewLines(String input) {
```

Comment:
almost duplicated with `escapeText`. Please let me know if you know better idea.

Reply:
Technically we can add another parameter to `escapeText` (similar to `camelize`) but that probably will require code updates in lots of places.

Let's go with this solution for the time being.